app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from __future__ import annotations
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players_list: list[Player]) -> int:
+    sum_rating = sum([player.get_rating() for player in players_list])
```

Comment:
```suggestion
    sum_rating = sum(player.get_rating() for player in players_list)
```
Convert to list is not needed